.github/workflows/isotovideo-check-all-test-modules.yml

```
@@ -13,4 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Run isotovideo against test code, fail if any test module failed
-        run: podman run --rm -it -v .:/tests:Z --entrypoint '' registry.opensuse.org/devel/openqa/containers/isotovideo:qemu-x86-jq /bin/sh -c 'isotovideo qemu_no_kvm=1 casedir=/tests && jq .result testresults/result-*.json | grep -v ok && echo "Test modules failed" && exit 1'
+        run: |
+          docker run --rm -v .:/tests:Z --entrypoint '' \
+          registry.opensuse.org/devel/openqa/containers/isotovideo:qemu-x86-jq /bin/sh \
+          -c 'isotovideo qemu_no_kvm=1 casedir=/tests \
+          && jq .result testresults/result-*.json | grep ok \
+          || (echo "Test modules failed" && exit 1)'
```

Same grep and subshell problem as above,
```suggestion
          && jq .result testresults/result-*.json | grep -v ok \
          && { echo "Test modules failed" && exit 1; } || true'
```